src/utils/HttpClient.ts

```diff
@@ -24,7 +24,12 @@ abstract class HttpClient {
 
 
   public authenticate = (): void => {
-    this.token = ''; // TODO: generate a valid token for authenticated methods
+    this.token = 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiUmFmYSIsInVzZXJJZCI6IjZqckhud2' +
+        'lIaWhSTmVXcXJLaXJXIiwicm9sZXMiOlsidXNlciJdLCJpYXQiOjE2NTIwMjE0MjAsImV4cCI6MjUxNjAyMTQy' +
+        'MH0.Vqt-nEOovaIiClBdJKcJMAk3WYN8-EmtCiwZ6YXU9qWjHgq0GHCEBHxnEclVG0n-gADWqN1A3lOtHk7a7F4lbNVT' +
+        'b5uKt4EJIaC48Rt3ePSEguaj6rmU5RyfnjRH_gkSEG1uCaff0Ox2IAuEU1lWHXenfoRtCevaSgcRRJm_-RH8J28QswQV9' +
+        '0dGKT9a6jUYDu8tf0dTvbEoazVPebgemTsgvlg8eC6QHO9ukh3fv1r561rQtDaDs685u7F7xbmPSocXhj_4MYq4ue_OB-qi' +
+        'DD6K8hxtH8fdzPnjUrjC1WufgmenGTAqhOsZrgFaKYaLMyH4Cm2oAxzlfES5j3BnIQ'; // TODO: generate a valid token for authenticated methods
```

Comment:
O que é que isto está aqui a fazer? Espero que não tenha nada a ver com esta tarefa = > https://trello.com/c/rElqyA82/20-gerar-um-token-para-os-testes-unit%C3%A1rios

As coisas que são "segredos" nunca podem ir parar a git... se isto fosse um token válido de uma ambiente de produção, acabarias de ter comprado uma carga de trabalhas e ias ser alvo de chacota de toda a empresa... depois de um token válido (e ainda por cima, que não expira) ter sido colocado em Github, o correcto seria:
1. terias de mudar o teu [JWT_SECRET](https://github.com/lopesnuno/chat/blob/master/src/config/index.ts#L51) do ambiente de produção, e 
2. forçar toda a gente a voltar a autenticar!

Como não é o caso, vais apagar isso daí, adicionar um token ao teu .env, alterar o (authenticate)[https://github.com/lopesnuno/chat/blob/master/src/utils/HttpClient.ts#L9] para receber como parâmetro um token e, chamar o `authenticate(token)` [aqui](https://github.com/lopesnuno/chat/blob/master/src/modules/rooms/RoomSpec.ts#L7)

Reply:
Ou algo do género!